services/movie_session.py

```diff
@@ -42,3 +43,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list:
+    tickets = Ticket.objects.filter(movie_session_id=movie_session_id)
```

Comment:
It would be better to use `.values("row", "seat")` in the query to obtain relevant results without the need for list comprehension. Rewrite this method using this approach.